packages/sdk/src/modules/JsonRpcDatasource.ts

```diff
@@ -0,0 +1,238 @@
+import { Transaction as BTCTransaction } from "bitcoinjs-lib";
+
+import { rpc } from "../api/jsonrpc";
+import type {
+  GetBalanceOptions,
+  GetInscriptionOptions,
+  GetInscriptionsOptions,
+  GetInscriptionUTXOOptions,
+  GetSpendablesOptions,
+  GetTxOptions,
+  GetUnspentsOptions,
+  GetUnspentsResponse,
+  RelayOptions,
+} from "../api/types";
+import type { Network } from "../config/types";
+import { OrditSDKError } from "../errors";
+import type { Inscription } from "../inscription/types";
+import type { Transaction, UTXO, UTXOLimited } from "../transactions/types";
+import { outpointToIdFormat } from "../utils";
+import { BaseDatasource } from "./BaseDatasource";
+import { DatasourceUtility } from "./DatasourceUtility";
+import type { JsonRpcPagination } from "./types";
+
+interface JsonRpcDatasourceOptions {
```

Comment:
can we export this as well, so any class that extends this `JsonRpcDatasource` can extend the option type as well

Reply:
Even better, I think we should remove this and re-use `BaseDatasourceOptions`.